lms/djangoapps/course_api/views.py

```diff
@@ -20,12 +26,13 @@ class CourseView(APIView):
     def get(self, request, course_key_string):
         """
         Request information on a course specified by `course_key_string`.
-            Body consists of a `blocks_url` that can be used to fetch the
-            blocks for the requested course.
+
+        Body consists of a `blocks_url` that can be used to fetch the blocks
+        for the requested course.
 
         Arguments:
             request (HttpRequest)
-            course_key_string
+            course_key_string: string representing the course key
```

Comment:
Nit: be consistent and put the data type (`string`) in parenthesis instead of in the description?